angr/knowledge_plugins/functions/function.py

```diff
@@ -1075,7 +1072,7 @@ def find_declaration(self):
             # PLT entries must have the same declaration as their jump targets
             # Try to determine which library this PLT entry will jump to
             edges = self.transition_graph.edges()
-            if len(edges) == 1 and type(next(iter(edges))[1]) is HookNode:
+            if len(edges) == 1 and isinstance(next(iter(edges))[1], HookNode):
```

Comment:
We use `type()` for speed issues. (`isinstance()` is significantly slower than `type()`).